tui_examples/shell_using_salome_GUI_model/README.md

```diff
@@ -0,0 +1,31 @@
+## Shell
+
+The example represents a simple shell model, fixed on one side and loaded on the other side.
+
+<img src="media/displacement.png" width="300">
+
+
+## Workflow
+
+This example shows how to use the KratosSalomePlugin to create a .mdpa file as a post-processing step to the graphical setup of the geometry and mesh in the Salome GUI.
+
+The geometry and mesh is already created and stored in the `plate_with_hole.hdf` Salome file.
+
+<img src="media/geometry.png" width="200">
+Geometry
+
+<img src="media/mesh.png" width="200">
+Mesh
+
+Using the `Entry` values of the Salome mesh objects, the .mdpa modelpart is set up in the `create_mdpa.py` script.
+
+<img src="media/object_browser.png" width="200">
+`Entry` values in the object browser
+
+
+### Instructions
+**Note:** The script is executed from the working directory of Salome. Therefor it is necessary to specify **absolute paths** to the KratosSalomePlugin and the desired output folder in the `create_mdpa.py` script.
+
+* Open the `plate_with_hole.hdf` in Salome
```

Comment:
can you please add the dumped python file too?
Such that it would be possible for newer versions of Salome in the future to recreate the example in case the hdf file cannot be read any more